app.py

```
@@ -87,6 +115,42 @@ def server(input, output, session: Session):
+        reactive.invalidate_later(5)
+
+        # 1. เช็คสถานะ Cache (L1)
+        cache_stats = COMPUTATION_CACHE.get_stats()
+        # ถ้ามีของใน Cache ให้เป็นเขียว 🟢 ถ้าว่างให้เป็นเทา ⚪
+        cache_icon = "🟢" if cache_stats['cached_items'] > 0 else "⚪"
+        cache_title = f"Cache: {cache_stats['cached_items']}/{cache_stats['max_size']} items (Hit rate: {cache_stats['hit_rate']})"
+
+        # 2. เช็คสถานะ Memory (L2)
+        mem_status = MEMORY_MANAGER.get_memory_status()
+        mem_icon = "💗" # ปกติ
+        if mem_status['status'] == 'WARNING': mem_icon = "💛" # เริ่มเยอะ
```

Ruff (0.14.10)

Multiple statements on one line (colon)

(E701)